README.md

```diff
@@ -128,6 +128,13 @@ To debug over Wi-Fi: http://blog.fossasia.org/android-app-debugging-over-wifi-fo
 Note :
 1. If you built your own hardware, change VendorID and/or ProductID in [CommunicationHandler.java](https://github.com/fossasia/pslab-android/blob/master/app/src/main/java/org/fossasia/pslab/communication/CommunicationHandler.java)
 
+### Permissions Required
+
+1. Record_Audio : It is required because in oscilloscope we are accepting inputs from the mic so we need to add this permission in order to contiune the proper functioning of this feature.You can find its implementation in [AudioJack.java](https://github.com/fossasia/pslab-android/blob/development/app/src/main/java/io/pslab/others/AudioJack.java).
+2. Access_Fine_Location And Internet : They are required because in compass and Map we need to get the location access so that the full use of these features can be taken.You can find its implementation in [GPSLogger.java](https://github.com/fossasia/pslab-android/blob/development/app/src/main/java/io/pslab/others/GPSLogger.java).
```

Comment:
1. Access_Fine_Location And Internet --> Access_Fine_Location and Internet
2. taken.You --> taken. You

Reply:
For use in lux meter and compass to get the coordinates for tagging the data on the map. You can find ....